File: system/src/main/resources/static/system/js/service/userService.js
app.service("userService", function ($http) {
    // 条件查询
    this.search = function (page, rows, searchEntity) {
        return $http.post(basePath + "/user/list?page=" + page + "&rows=" + rows,
            searchEntity);
    };
    // 禁用
    this.disable = function (selectIds) {
        return $http.delete(basePath + "/user/nouse?ids=" + selectIds);
    };

    //查询当前用户资料
    this.loadUserInfo = function () {
        return $http.get(basePath + "/user/load");
    };

    this.modifyPassword = function (info) {
        return $http.post(basePath + "/user/modifyPassword",info);
    };
});

File: system/src/main/java/com/cys/system/common/service/UserService.java
package com.cys.system.common.service;

import com.cys.system.common.common.pojo.Result;
import com.cys.system.common.pojo.User;

public interface UserService {
    Result listUser(Integer page, Integer rows, User user);

    Result nouse(Integer[] ids);

    Result load(Integer userId);

    Result modifyPassword(Integer userId,String passwordInfo);
}
